wordpress-mobile/WordPress-iOS
# [Stats Refresh] Update Countries Map colours

Refs. #11852

This PR updates the colours used in the Countries Map.

![Simulator Screen Shot - iPhone X - 2019-07-02 at 13 28 57](https://user-images.githubusercontent.com/912252/60514890-b9788e00-9cda-11e9-86c8-67f533032f46.png)


## To test:
• Open Stats -> DWMY Period (Use a site with data)
• Scroll until the map is displayed
• Be sure the map uses the pink colour.

Update release notes:
